backend/src/generated/tmp-resolvers/Mutation.ts

```diff
@@ -0,0 +1,14 @@
+// This resolver file was scaffolded by github.com/prisma/graphqlgen, DO NOT EDIT.
+// Please do not import this file directly but copy & paste to your application code.
+
+import { MutationResolvers } from "../graphqlgen";
+
+export const Mutation: MutationResolvers.Type = {
+  ...MutationResolvers.defaultResolvers,
+  write: (parent, args, ctx) => {
+    throw new Error("Resolver not implemented");
+  },
+  upvote: (parent, args, ctx) => {
+    throw new Error("Resolver not implemented");
+  }
+};
```

Comment:
your folders and files are very nested and you created a quite big folder tree. We don't really get why you made an extra file for Resolver Mutations.

Reply:
what exactly are you doing in the write / upvote Mutation? seems like there is just an Error thrown ?! or are we wrong?